components/MainNavbar.tsx

```diff
@@ -1,65 +1,96 @@
-      {/* <Navbar.Brand>
-        <Navbar.Toggle aria-label="toggle navigation" showIn="xs" />
-        <Image src={terraLogo} alt="Terra Logo" width={50} height={50} />
-        <Text b color="inherit" hideIn="xs">
-          Terra
-        </Text>
-      </Navbar.Brand> */}
-      <Navbar.Content hideIn="xs">
-        {/* TODO: We need an actual docs page */}
-        <Navbar.Link href="https://github.com/terrapkg/packages/tree/main/docs">
-          Docs
-        </Navbar.Link>
-        <Navbar.Link href="https://github.com/terrapkg/packages">
-          Sources
-        </Navbar.Link>
-        {/* <Navbar.Link href="#">Search</Navbar.Link> */}
-      </Navbar.Content>
-      <Navbar.Collapse>
-        <Navbar.CollapseItem>
-          <Link
-            href="https://github.com/terrapkg/packages/tree/main/docs"
-            color="inherit"
-            css={{
-              minWidth: "100%",
-            }}
-          >
+    <>
+      <InstallModal
+        visible={showInstallModal}
+        closeHandler={() => setShowInstallModal(false)}
+      />
+      <Navbar variant="static" maxWidth="fluid">
+        <Navbar.Brand>
+          <Navbar.Toggle aria-label="toggle navigation" showIn="xs" />
+          <Image src={terraLogo} alt="Terra Logo" width={50} height={50} />
+          <Text color="inherit" weight="bold">
+            Terra
+          </Text>
+        </Navbar.Brand>
+        <Navbar.Content hideIn="xs">
+          <Navbar.Link href="https://github.com/terrapkg/packages/wiki">
             Docs
-          </Link>
-        </Navbar.CollapseItem>
-        <Navbar.CollapseItem>
-          <Link
-            href="https://github.com/terrapkg/packages"
-            color="inherit"
-            css={{
-              minWidth: "100%",
-            }}
-          >
+          </Navbar.Link>
+          <Navbar.Link href="https://github.com/terrapkg/packages">
             Sources
-          </Link>
-        </Navbar.CollapseItem>
-        {/* <Navbar.CollapseItem>
-          <Link
-            href="#"
-            color="inherit"
-            css={{
-              minWidth: "100%",
-            }}
-          >
-            Search
-          </Link>
-        </Navbar.CollapseItem> */}
-      </Navbar.Collapse>
-    </Navbar>
+          </Navbar.Link>
+          <Navbar.Link href="https://repology.org/repository/terra_37">
+            Repology
+          </Navbar.Link>
+          {/* <Navbar.Link href="https://github.com/terrapkg/packages/tree/main/docs">
+            Explore
+          </Navbar.Link> */}
+        </Navbar.Content>
+        <Navbar.Content>
+          <Navbar.Item>
+            <Button auto flat onClick={() => setShowInstallModal(true)}>
+              Install
+            </Button>
+          </Navbar.Item>
+        </Navbar.Content>
+        <Navbar.Collapse>
+          <Navbar.CollapseItem>
+            <Link
+              href="https://github.com/terrapkg/packages/tree/main/docs"
```

Comment:
docs folder doesn't exist anymore

Reply:
Link this to wiki + instead of calling this "About" just make it consistent with the desktop version